WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1114,11 +1114,10 @@ public void onEventMainThread(StorySaveEvents.StorySaveResult event) {
                                         event.getStoryIndex()) + ""
                         );
 
-                        // TODO WPSTORIES add TRACKS: the putExtra described here below for NOTIFICATION_TYPE
-                        // is meant to be used for tracking purposes. Use it!
-                        // TODO add NotificationType.MEDIA_SAVE_ERROR param later when integrating with WPAndroid
-                        //        val notificationType = NotificationType.MEDIA_SAVE_ERROR
-                        //        notificationIntent.putExtra(ARG_NOTIFICATION_TYPE, notificationType)
+                        mStoriesTrackerHelper.trackStorySaveResultEvent(
+                                event,
+                                Stat.STORY_SAVE_ERROR_SNACKBAR_MANAGE_TAPPED
```

Comment:
ended up adding a new event for the specific case of tapping on the `manage` action on the error snackbar, given `NOTIFICATION_TYPE`  is reserved only for the case when the user taps on the error notification on the system dashboard